core/src/test/java/org/sql2o/converters/LocalDateTimeConverterTest.java

```diff
@@ -0,0 +1,189 @@
+        try (Connection connection = sql2o.open(); Query query = connection.createQuery(sqlScriptOfInsertData)) {
+
+            query.addParameter("created_at", LocalDateTime.now());
+            query.executeUpdate();
+        }
+    }
+
+
+    @After
+    public void tearDown() {
+        this.dropTable();
+    }
+
+    private void dropTable() {
+        String sqlScriptOfDropTable = "drop table local_date_time_converter";
+        try (Connection connection = sql2o.open(); Query query = connection.createQuery(sqlScriptOfDropTable)) {
+
+            query.executeUpdate();
+        }
+    }
+
+
+    @Test
+    public void whenFindAllLocalDateTimes_thenTableHas1InitialRecordsOfLocalDateTimeInstance() {
+
+        // When
+        List<LocalDateTime> localDateTimes;
+        String sqlScriptOfFindAll = "select created_at from local_date_time_converter";
+        try (Connection connection = sql2o.open(); Query query = connection.createQuery(sqlScriptOfFindAll)) {
+
+            List<MockLocalDateTime> mockLocalDateTimes = query.executeAndFetch(MockLocalDateTime.class);
+            localDateTimes = mockLocalDateTimes.stream()
+                .map(MockLocalDateTime::getLocalDateTime)
+                .collect(Collectors.toList());
+        }
+
+        // Then
+        Assert.assertNotNull(localDateTimes);
+        Assert.assertEquals(1, localDateTimes.size());
+        Assert.assertEquals(LocalDateTime.class, localDateTimes.get(0).getClass());
+    }
+
+
+    @Test
+    public void givenValidLocalDateTime_whenLocalDateTimeSavedSuccessfully_thenTableHas2RecordsOfLocalDateTimeInstance() {
+
+        // Given
+        LocalDateTime localDateTime = LocalDateTime.now();
+        String sqlScriptOfInsertData = "insert into local_date_time_converter(created_at) values (:created_at)";
+        try (Connection connection = sql2o.open(); Query query = connection.createQuery(sqlScriptOfInsertData)) {
+            query
+                .addParameter("created_at", localDateTime)
+                .executeUpdate();
+        }
+
+        // When
+        List<LocalDateTime> localDateTimes;
+        String sqlScriptOfFindAll = "select created_at from local_date_time_converter";
+        try (Connection connection = sql2o.open(); Query query = connection.createQuery(sqlScriptOfFindAll)) {
+
+            List<MockLocalDateTime> mockLocalDateTimes = query.executeAndFetch(MockLocalDateTime.class);
+
+            localDateTimes = mockLocalDateTimes.stream()
+                .map(MockLocalDateTime::getLocalDateTime)
+                .collect(Collectors.toList());
+        }
+
+        // Then
+        Assert.assertNotNull(localDateTimes);
+        Assert.assertEquals(2, localDateTimes.size());
+        Assert.assertEquals(LocalDateTime.class, localDateTimes.get(0).getClass());
+        Assert.assertEquals(LocalDateTime.class, localDateTimes.get(1).getClass());
+    }
```

Comment:
Same comment as above.

Reply:
Thank you for your feedback.

The main purpose of checking two records here is to also verify the record initially added to the table by default. Because we don't know whether the upcoming data is new or old. That's why we are checking both records here.

However, if you have any other concerns or suggestions regarding the test coverage, I'm open to discussing them.